optuna/logging.py

```diff
@@ -1,12 +1,14 @@
 import colorlog
 import logging
+
```

Comment:
We usually cluster standard library imports in a single island. Could you undo this change?
```suggestion
```